app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -8,32 +8,40 @@ class ReferenceWithFeedbackComponent < ViewComponent::Base
              :relationship,
              :feedback_status,
              :consent_to_be_contacted,
+             :relationship_confirmation,
+             :relationship_correction,
+             :referee_type,
              to: :reference
 
     def initialize(reference:, reference_number:, editable:)
       @reference = reference
       @reference_number = reference_number
+      @ordinal = TextOrdinalizer.call(reference_number)
```

Comment:
Cool!

Reply:
(Copied and pasted from the Manage interface 😄 )

Reply:
Nice, will remember this!